charts/cloud-baseline/values.yaml

```diff
@@ -13,6 +13,9 @@ traefik:
       port: 80
       nodePort: 32414
       targetPort: web
+  providers:
+  kubernetesCRD:
+    allowCrossNamespace: true
```

Comment:
this shouldn't be on the cloud should it? We would be using EFS for cloud

Reply:
This is nothing to do with EFS. This is allowing cross namespace communication